node/nodeRunner.go

```diff
@@ -724,6 +735,10 @@ func waitForSignal(
 	if reshuffled {
 		log.Info("=============================" + SoftRestartMessage + "==================================")
 		core.DumpGoRoutinesToLog(goRoutinesNumberStart)
+		err := oldDbCleaner.CleanOldStorageForShuffleOut(oldDbCleanerArgs)
+		if err != nil {
+			log.Warn("cannot clean old db", "error", err)
+		}
```

Comment:
no, I can't, because the `else` branch is for non-shuffle-out reasons. and adding a return here is useless because it will by anyway the next executed line

Reply:
please read again my comment, this is about restructuring the code so you don't need an else branch